ui/packages/catalog-ui-search/src/main/webapp/component/confirmation/confirmation.less

```diff
@@ -14,20 +14,35 @@
     left: ~'calc(50% - 200px)';
     top: 50%;
     height: auto;
-    width: 400px;
+    width: 500px;
```

Comment:
@andrewkfiedler @adimka Do we need to multiply this by a factor of X with some less variable or only for smaller values?